.github/workflows/example-tvos-build-check.yml

```diff
@@ -42,51 +43,20 @@ jobs:
       - name: Install monorepo node dependencies
         run: yarn install --immutable
 
-      - name: Restore Reanimated node_modules from cache
```

Comment:
Let's keep such comment here instead:

`# TODO: Add caching for node_modules and artifacts that will work with monorepo setup.`